flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/DataStreamUtils.java

```diff
@@ -122,7 +122,7 @@
         final ClientAndIterator<E> clientAndIterator = collectWithClient(stream, jobName);
         final List<E> result = collectRecordsFromUnboundedStream(clientAndIterator, numElements);
 
-        // cancel the job not that we have received enough elements
+        // cancel the job note that we have received enough elements
```

Comment:
I doubt that this is what was meant here

Reply:
But  should  'not' to be removed ?   I think the meaning is ,   we have received enough elements, so we can cancel the job ?

Reply:
I hope that when a PR is submitted for a fix, that the contributor knows what's meant in this case. Else, how do you that the PR is an improvement? So that means checking out what the test does and fixing (if applicable) the comment.

Do note that this is a comment typo fix only: that means that the impact and value of this PR is limited because it the impact for the Flink user or developer is super small.

Reply:
I have tried my best to understand the code corresponding to the comment. Before submitting the pr, I do think this is an improvement. If there is something wrong with me, I will be more careful next time.

Reply:
I think what was meant was `cancel the job now that we have received enough elements`.

Reply:
Forgiving my pool english，I'll change the comment too. Thanks for your patience.